### ISSUE TITLE
POIs shown by default

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/maps#13]
Currently when clicking »Amenity« or »Tourism«, all the layers are shown. This is way too much. Instead we should go the other way around by not showing all but instead only a few:

By default enabled for Amenity should be
* toilets
* drinking_water
* atm
* hospital
* post_office

Tourism:
* attraction
* museum
* viewpoint

For the shops … wow – we really simply need one unified search/filter box (like Google Maps for amenities as well as addresses).

**Comment by poVoq** [comment]
I think there should be an option to disable these all together.

**Comment by v1r0x** [comment]
If we use the maki icons instead of real markers  (#52) we could display them depending on the current zoom level. E.g. if I zoom out to see the map of a city there should be only the most important ones, but as soon as I zoom in there should be more. The more I zoom in the more markers should be displayed. I already discussed it with @jancborchardt but one of our concerns was the possible impact on the performance.
Maybe this could be an alternative to the bunch of togglable markers we have right now (of course only if there is no/small impact on the performance)?

cc @Henni @DJaeger

**Comment by jancborchardt** [comment]
Yeah, I guess basically this is how Google Maps does it – you can just click restaurants, cafés etc. We only have a static map, where you can long click some elements and hope it’s a POI, or toggle the POI layers. Showing them always will be a much better experience.

**Comment by v1r0x** [comment]
I'll close this in favor of #120
@jancborchardt reopen this if you think we still need it.

**Comment by jancborchardt** [comment]
We should still have some categories by default which highlight certain things. Like restaurants, cafes, wifi, ATM etc. Just a few things. Lots of mobile maps apps do this and it’s quite useful. Moving to 0.2 though.

**Comment by v1r0x** [comment]
In the current master we show all markers, unless the user hides the POI layer. But depending on the zoom level we show/hide some of the POIs. Thought that's what we wanted?

**Comment by jancborchardt** [comment]
Yeah, that is of course awesome. I was just thinking that maybe a way to highlight an entire category of markers would be cool. Like you’re looking for a restaurant nearby, or cool wifi spots in a city you’re visiting etc. It’s a bit difficult to look for the specific markers, or type in »Restaurant« to the search.

Essentially of course this function would just be a shortcut to typing »wifi« or »restaurant« in the search. But that’s super useful. :) Compare it with Maps.me on Android or Google Maps as well.

**Comment by v1r0x** [comment]
I'm not quite sure what you mean. Do you want to set a different color for each POI type (wifi blue, restaurant red, ...) or sublayers in the nav bar?

**Comment by jancborchardt** [comment]
The basic idea would be for example: When you click on »wifi« in that specific places list to highlight all the wifi places in the view. This could be done by colorization of the icon (if that’s easily possible), or placing a marker on every relevant place. Similarly to how we do it when you would put »restaurant« in the search box.

**Comment by v1r0x** [comment]
Colorization should be easily possible. We currently have a basic POI icon css class. We could create a couple more for wifi, etc and change the background color. But I'm still not completely convinced. This would extend the nav bar, which should be as clean as possible.
Typing a poi type in the search field doesn't work with the new geocoder. I think the way google maps does it, is a good solution. Searching for "restaurant" highlights them with a background color. We could also think about adding a "search for 'restaurant' in your area..." as first item in the result list.

**Comment by jancborchardt** [comment]
Yeah, let’s put this in the backlog for now. I don’t want to completely close the idea because even though desktop Maps apps don’t do this, basically all mobile ones do – as shortcuts for typing.
